layouts/_default/baseof.html

```diff
@@ -60,9 +60,13 @@
     {{ end }}
 
     {{ range .Site.Params.custom_css }}
-      <link rel="stylesheet" href="{{ . | relURL }}">
+      <link rel="stylesheet" href="{{ . | relURL }}"/>
```

Comment:
```suggestion
      <link rel="stylesheet" href="{{ . | relURL }}" />
```
Just adding a space here.